ts/features/zendesk/screens/ZendeskAskSeeReportsPermissions.tsx

```diff
@@ -0,0 +1,164 @@
+import { Link } from "../../../components/core/typography/Link";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../components/ui/FooterWithButtons";
+import { zendeskPrivacyUrl } from "../../../config";
+import I18n from "../../../i18n";
+import {
+  AppParamsList,
+  IOStackNavigationProp,
+  IOStackNavigationRouteProps
+} from "../../../navigation/params/AppParamsList";
+import { useIOSelector } from "../../../store/hooks";
+import {
+  profileEmailSelector,
+  profileFiscalCodeSelector,
+  profileNameSurnameSelector
+} from "../../../store/reducers/profile";
+import { showToast } from "../../../utils/showToast";
+import { openWebUrl } from "../../../utils/url";
+import ZendeskItemPermissionComponent, {
+  ItemPermissionProps
+} from "../components/ZendeskItemPermissionComponent";
+import { ZendeskParamsList } from "../navigation/params";
+
+export type ZendeskAskSeeReportsPermissionsNavigationParams = {
+  assistanceForPayment: boolean;
+};
+
+type ItemProps = {
+  fiscalCode: string;
+  nameSurname: string;
+  email: string;
+};
+
+const iconProps = { width: 24, height: 24 };
+
+const getItems = (props: ItemProps): ReadonlyArray<ItemPermissionProps> => [
+  {
+    id: "profileNameSurname",
+    icon: <NameSurnameIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.nameSurname"),
+    value: props.nameSurname,
+    testId: "profileNameSurname"
+  },
+  {
+    id: "profileFiscalCode",
+    icon: <FiscalCodeIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.fiscalCode"),
+    value: props.fiscalCode,
+    testId: "profileFiscalCode"
+  },
+  {
+    id: "profileEmail",
+    icon: <EmailIcon {...iconProps} />,
+    title: I18n.t("support.askPermissions.emailAddress"),
+    value: props.email,
+    testId: "profileEmail"
+  }
+];
+
+type Props = IOStackNavigationRouteProps<
+  ZendeskParamsList,
+  "ZENDESK_ASK_SEE_REPORTS_PERMISSIONS"
+>;
+
+/**
+ * this screen shows the kinds of data the app could collect when a user is looking at the previous ticket only for the authenticated users
+ * @constructor
+ */
+const ZendeskAskSeeReportsPermissions = (props: Props) => {
+  const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
+  const assistanceForPayment = props.route.params.assistanceForPayment;
+  const notAvailable = I18n.t("global.remoteStates.notAvailable");
+  const fiscalCode = useIOSelector(profileFiscalCodeSelector) ?? notAvailable;
+  const nameSurname = useIOSelector(profileNameSurnameSelector) ?? notAvailable;
+  const email = pipe(
+    useIOSelector(profileEmailSelector),
+    O.getOrElse(() => notAvailable)
+  );
+
+  const itemsProps: ItemProps = {
+    fiscalCode,
+    nameSurname,
+    email
+  };
+
+  const items = getItems(itemsProps)
+    // remove these item whose have no value associated
+    .filter(it => it.value !== notAvailable);
```

Comment:
Maybe we can filter the list without use `notAvailable` "placeholder".

```suggestion
  const { assistanceForPayment } = props.route.params;
  const fiscalCode = useIOSelector(profileFiscalCodeSelector);
  const nameSurname = useIOSelector(profileNameSurnameSelector);
  const email = pipe(
    useIOSelector(profileEmailSelector),
    O.fold(
      () => undefined,
      e => e
    )
  );

  const itemsProps: ItemProps = {
    fiscalCode,
    nameSurname,
    email
  };

  // remove these item whose have no value associated
  const items = getItems(itemsProps).filter(it => it.value);
```

Moreover for this suggestion we need to update `ItemProps` with optional properties.